### ISSUE TITLE
In 768-pixel screen size, the header menu design breaks. 

### ISSUE THREAD

**Issue by aezazs-multidots** [WordPress/Learn#1365]
## Description
In 768-pixel screen size, the header menu design breaks. 

## Step-by-step reproduction instructions
Open [this](https://learn.wordpress.org/tutorial/using-the-wordpress-rest-api/) page in 768-pixel resolution 

## Screenshots or screen recording (optional)
![CleanShot 2023-02-22 at 11 59 11@2x](https://user-images.githubusercontent.com/72435501/220540887-53ae6994-d30c-4647-9025-6b4b00f6baaf.png)

**Comment by jonathanbossenger** [comment]
This is waiting on the PR to be refreshed.

**Comment by rjekic** [comment]
This issue is no longer relevant, as the site's design has changed. The issue is closed, and can be reopened if there is a need.

cc @devmuhib009 @WordPress/learn-editors
